src/style.css

```diff
@@ -0,0 +1,126 @@
+* {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  width: 198px;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  padding-top: 32px;
+  padding-left: 19px;
+  padding-right: 19px;
+  padding-bottom: 40px;
```

Comment:
combine in just one `padding` property, move this into `.card`. fix in other places